docs/content/latest/explore/indexes-constraints/constraints.md

```diff
@@ -0,0 +1,298 @@
+
+Most commonly, the primary key is added to the table when the table is created, as demonstrated by the following syntax:
+
+```
+CREATE TABLE (
+  column1 data_type PRIMARY KEY,
+  column2 data_type,
+  column3 data_type,
+  …
+);
+```
+
+Suppose you work with a database that requires the following table to be created:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text
+);
+```
+
+The primary key of the `employees` table is `employee_no`, which uniquely identifies an employee.
+
+The following syntax the primary key definition for more than one column:
+
+```
+CREATE TABLE (
+  column1 data_type,
+  column2 data_type,
+  column3 data_type,
+  …
+  PRIMARY KEY (column1, column2)
+);
+```
+
+The following example creates the `employees` table in which the primary key is a combination of `employee_no` and `name`:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer,
+  name text,
+  department text,
+  PRIMARY KEY (employee_no, name)
+);
+```
+
+YSQL assigns a default name in the format `tablename_pkey` to the primary key constraint. In the preceding example, it is `employees_pkey`. If you need a different name, you can specify it using the `CONSTRAINT` clause, as per the following syntax:
+
+```
+CONSTRAINT constraint_name PRIMARY KEY(column1, column2, ...);
+```
+
+In some cases you might decide to define a primary key for an existing table. To do this, you use the `ALTER TABLE` statement, as per the following syntax:
+
+```
+ALTER TABLE table_name ADD PRIMARY KEY (column1, column2);
+```
+
+The `ALTER TABLE` statement also allows you to add an auto-incremented primary key to an existing table by using the [SERIAL type](https://docs.yugabyte.com/latest/explore/ysql-language-features/data-types/#serial-pseudotype), as per the following syntax:
+
+```
+ALTER TABLE table_name ADD COLUMN ID SERIAL PRIMARY KEY;
+```
+
+You can remove a primary key constraint by using the `ALTER TABLE` statement, as demonstrated by the following syntax:
+
+```
+ALTER TABLE table_name DROP CONSTRAINT primary_key_constraint;
+```
+
+The following example shows how to remove the primary key constraint from the `employees` table:
+
+```
+ALTER TABLE employees DROP CONSTRAINT employees_pkey;
+```
+
+For more information and examples, see the following: 
+
+- [Primary Key](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_table/#primary-key)
+- [Table with Primary Key](https://docs.yugabyte.com/latest/api/ysql/the-sql-language/statements/ddl_create_table/#table-with-primary-key)
+- [Primary Keys in PostgreSQL documentation](https://www.postgresql.org/docs/12/ddl-constraints.html#DDL-CONSTRAINTS-PRIMARY-KEYS)
+
+## Foreign Key
+
+A foreign key represents one or more columns in a table referencing the primary key of another table.
```

Comment:
I think it can reference either a primary key or unique index (i.e. columns that are restricted with unique index or unique constraint) but let's confirm that by testing.

Reply:
Yes, you are right, I tested both the unique index and unique constraint, and the both scenarios work. I'm adding this info to the document.